### PR TITLE
Update to Bevy 0.14

### PR DESCRIPTION
**Pull request by Friz64**
Closes #280

### CODE REVIEW

**Comment by atbentley** [comment]
@Friz64 could we bump this PR to the 0.14 release candidate?

```diff
diff --git a/Cargo.toml b/Cargo.toml
index 07803dd..6aff021 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,7 +39,7 @@ name = "ui"
 required-features = ["render"]
 
 [dependencies]
-bevy = { git = "https://github.com/bevyengine/bevy.git", rev = "519abbc", default-features = false, features = [
+bevy = { version = "0.14.0-rc.2", default-features = false, features = [
     "bevy_asset",
 ] }
 egui = { version = "0.27", default-features = false, features = ["bytemuck"] }
@@ -52,7 +52,7 @@ thread_local = { version = "1.1.0", optional = true }
 
 [dev-dependencies]
 version-sync = "0.9.4"
-bevy = { git = "https://github.com/bevyengine/bevy.git", rev = "519abbc", default-features = false, features = [
+bevy = { version = "0.14.0-rc.2", default-features = false, features = [
     "x11",
     "png",
     "bevy_pbr",
```

**Comment by Friz64** [comment]
Yes, I have that in mind for later today, when I am able to.

**Comment by tychedelia** [comment]
Thanks for doing this!

File: Cargo.toml [hunk 1]
```diff
@@ -39,8 +39,11 @@ name = "ui"
 required-features = ["render"]
 
 [dependencies]
-bevy = { version = "0.13", default-features = false, features = ["bevy_asset"] }
+bevy = { version = "0.14.0-rc.2", default-features = false, features = [
```
Comment (MiniaczQ) [inline Cargo.toml:42]:
```suggestion
bevy = { version = "0.14.0-rc.3", default-features = false, features = [
```

File: Cargo.toml [hunk 2]
```diff
@@ -49,7 +52,7 @@ thread_local = { version = "1.1.0", optional = true }
 
 [dev-dependencies]
 version-sync = "0.9.4"
-bevy = { version = "0.13", default-features = false, features = [
+bevy = { version = "0.14.0-rc.2", default-features = false, features = [
```
Comment (MiniaczQ) [inline Cargo.toml:55]:
```suggestion
bevy = { version = "0.14.0-rc.3", default-features = false, features = [
```

File: Cargo.toml (hunk 1 above)
Comment (Friz64) [reply Cargo.toml]: This is not necessarily needed, a `cargo update` will also get you onto `rc.3`, but this won't hurt.

**Comment by xangelix** [comment]
Bump this to 0.14.0 now that it's released! Thanks :)

**Comment by Friz64** [comment]
There you go!!! 🎉

**Comment by mvlabat** [comment]
The colours seem to be wrong on WASM. I won't be blocking the release, but if anyone could help me with fixing those, that would be great.

And thank you for the PR @Friz64!

**Pull request merged**